lib/smooth/application/display/DisplaySpi.cpp

```diff
@@ -154,17 +212,17 @@ namespace smooth::application::display
         return res;
     }
 
-    // Send data to the LCD. Uses spi_device_polling_write, which waits until the
-    // transfer is complete.
-    bool ILI9341::lcd_wr_data(const uint8_t* data, size_t length)
+    // Send data to the displayc.
+    // Uses spi_device_polling_write, which waits until the transfer is complete.
+    bool DisplaySpi::send_data(const uint8_t* data, size_t length)
```

Comment:
Do you have plans to make this API async, i.e. non-blocking?

Reply:
Removed comment

Reply:
Nothing wrong with the comment, I was just wondering :)